sdk/core/Azure.Core/api/Azure.Core.net461.cs

```diff
@@ -530,7 +534,7 @@ public static partial class MultipartResponse
     {
         private readonly object _dummy;
         private readonly int _dummyPrimitive;
-        public string? Id { get { throw null; } }
+        public string Id { get { throw null; } }
```

Comment:
I agree with this change -- on the basis that if Operation.Id is equivalent to RehydrationToken.Id, they should both have the same type.  I believe, though, that it is a breaking change, and because of this you may need to send email to the Azure SDK arch board to get approval for it.  You can check with @KrzysztofCwalina to see if this is needed.

Reply:
It's a get-only property. Why would this be a breaking change?